src/pages/library/dataBrowser-utils.js

```diff
@@ -15,6 +15,35 @@ export const snapshotAccessTypes = {
   PENDING: 'Pending'
 }
 
+/*
+  It's necessary to add the release policy prefix because of how the filters work.
+  Filters will grab from an array of values without differentiating which filter it belongs to
+
+  For example:
+  if the user decides to filter by "data release policy = 'other'", it will filter the data
+  based off "{any filter field} = 'other'". Adding the prefix forces the filtering to recognize
+  which field is being filtered on.
+*/
+const releasePolicyPrefix = 'releasepolicytag_'
+export const snapshotReleasePolicies = {
+  'TerraCore:NoRestriction': { tag: `${releasePolicyPrefix}NRES`, label: 'NRES', desc: 'No restrictions' },
```

Comment:
If we go with the above suggestions I think we can remove the `tag` prop, but I may be missing something.